README.md

```diff
@@ -2,162 +2,102 @@
+cd brapi
 ```
 
-The shiny-server.conf file is used to increase the default timeout to avoid errors on initialization if BETYdb is taking time to load.
-
-### Database Connection
+Within the brapi folder, create a `docker-compose.override.yml` file to expose a port on your local machine. The following chunk can be used to expose port 5432 and should be added to the file:
 
-This application requires a connection to a BETYdb database. It requires a BETYdb database that uses the experiments table to associate time ranges and sites or plots with specific experiments or seasons.
-
-Connection parameters can be set as environment variables. By default they are:
-
-```
-bety_dbname   = bety
-bety_password = SECRET
-bety_host     = bety6.ncsa.illinois.edu
-bety_port     = 5432
-bety_user     = viewer
+```sh
+version: '3'
+services:
+  postgres:
+    ports:
+      - '5432:5432'
 ```
 
-There are three methods that you can use access to the TERRAREF instance of BETYdb:
-
-1. If running on the NCSA Nebula cluster within the terraref domain, can be accessed by setting the environment variable `bety_host=bety6`
-    1. This can be done within the NDS Workbench at terraref.ndslabs.org
-2. Otherwise, use an ssh tunnel. Developers can get ssh access to the TERRA REF database server, you can [request here](https://identity.ncsa.illinois.edu/join/TU49BUUEDM) and the connect with
-    ```sh
-    ssh -Nf -L 5432:localhost:5432 bety6.ncsa.illinois.edu
-    ```    
-3. Install a local instance of BETYdb
-    ```sh
-    load.bety.sh -c -u -m 99 -r 0 -w https://terraref.ncsa.illinois.edu/bety/dump/bety0/bety.tar.gz
-    load.bety.sh -m 99 -r 6 -w https://terraref.ncsa.illinois.edu/bety/dump/bety6/bety.tar.gz
-    ```
-
-#### Deploy Updates on Nebula / Kubernetes
+Note: If port 5432 is being used on your computer, you can use any other number. Just make sure to change the first number in the pairing to the number you choose.
 
-After updating the master branch on GitHub, a new Docker image is built on [Docker hub](https://hub.docker.com/r/terraref/traitvis-webapp/builds/). The following commands can be used to deploy and inspect this app as it is being served at traitvis.workbench.org
-
-Login to server (requires authentication)
+Initialize the database:
 
 ```sh
-ssh core@282.284.420.38 # = ip x 2
+docker-compose up -d postgres
+docker-compose run --rm bety initialize
+docker-compose run --rm bety sync
 ```
 
-See what is running in the terraref namespace, including extractors and traitvis
-
-```sh
-kubectl get pods --namespace=terraref
-``` 
+#### R packages
 
+This application requires the following R packages:
 
-The following restarts the container, which triggers the most recent image from [docker hub](https://hub.docker.com/r/terraref/traitvis-webapp/builds/) to be downloaded. 
+`shiny`, `shinythemes`, `scales`, `lubridate`, `dplyr`, `ggplot2`, `timevis`, `rgeos`, `leaflet`, `cronR`, `stringr`, `kableExtra`, `raster`, `mapview`, `leafem`, and `sf`.
```

Comment:
Probably easier to list them at the same time that you show how to install them:

```r
install.packages(c('shiny', 'shinythemes', 'lubridate', 'dplyr', 'ggplot2', 'timevis', 'rgeos', 'leaflet', 'cronR', 
                                'stringr', 'kableExtra', 'raster', 'mapview', 'leafem', 'sf'))
```